orangecontrib/text/tests/test_preprocess.py

```diff
@@ -1,381 +1,226 @@
 import unittest
 
-from nltk.tokenize import word_tokenize, RegexpTokenizer, TweetTokenizer
+import nltk
 
-from orangecontrib.text.corpus import Corpus
-from orangecontrib.text.preprocess import (Preprocessor, PorterStemmer,
-                                           SnowballStemmer, Lemmatizer)
+from orangecontrib.text import preprocess
+from orangecontrib.text.preprocess import Preprocessor
 
 
 class PreprocessTests(unittest.TestCase):
-    TEST_STRING = 'Human machine interface for lab abc computer applications'
-    TEST_LIST = None
-    TEST_CORPUS = None
-
-    def setUp(self):
-        self.TEST_CORPUS = Corpus.from_file('deerwester')
-        self.TEST_LIST = [entry.metas[0] for entry in self.TEST_CORPUS]
-
-    def test_tokenize(self):
-        correct = [
-            ['human', 'machine', 'interface', 'for', 'lab', 'abc',
-             'computer', 'applications'],
-            ['a', 'survey', 'of', 'user', 'opinion', 'of', 'computer',
-             'system', 'response', 'time'],
-            ['the', 'eps', 'user', 'interface', 'management', 'system'],
-            ['system', 'and', 'human', 'system', 'engineering', 'testing',
-             'of', 'eps'],
-            ['relation', 'of', 'user', 'perceived', 'response', 'time',
-             'to', 'error', 'measurement'],
-            ['the', 'generation', 'of', 'random', 'binary', 'unordered',
-             'trees'],
-            ['the', 'intersection', 'graph', 'of', 'paths', 'in', 'trees'],
-            ['graph', 'minors', 'iv', 'widths', 'of', 'trees', 'and',
-             'well', 'quasi', 'ordering'],
-            ['graph', 'minors', 'a', 'survey']
-        ]
-
-        # String.
-        p = Preprocessor(lowercase=False)
-        self.assertEqual(
-            p(self.TEST_STRING),
-            [
-                [
-                    'Human',
-                    'machine',
-                    'interface',
-                    'for',
-                    'lab',
-                    'abc',
-                    'computer',
-                    'applications',
-                ]
-            ]
-        )
-
-        # List.
+    sentence = "Human machine interface for lab abc computer applications"
+    corpus = [
```

Comment:
Could we just load the `deerwester` corpus, like `Corpus.from_file('deerwester')` and use that?